### PR TITLE
Updates to EMAX and exponential models

### PR DESCRIPTION
**Pull request by mitchell-thomann**
Updated parameters for EMAX and exponential models including:

- JAGS code
- Documentation
- Posterior test cases
- Inform messages in model.R

### CODE REVIEW

File: R/dreamer_mean.R [hunk 1]
```diff
@@ -15,11 +15,12 @@ dreamer_mean_logquad <- function(dose, b1, b2, b3) {
 }
 
 dreamer_mean_emax <- function(dose, b1, b2, b3, b4) {
-  b1 + (b2 - b1) / (exp(b4 * (b3 - log(dose))) + 1)
+  b1 + b2 * dose ^ b4 / (b3 ^ b4 + dose ^ b4)
 }
 
 dreamer_mean_exp <- function(dose, b1, b2, b3) {
-  b1 + b2 * (1 - exp(- b3 * dose))
+  #b1 + b2 * (1 - exp(- b3 * dose))
```
Comment (rich-payne) [inline R/dreamer_mean.R:22]: remove commented code

File: R/model.R [hunk 2]
```diff
@@ -212,17 +212,22 @@ model_emax <- function(
   )
   class(mod) <- c("dreamer_emax", "dreamer_continuous")
   attr(mod, "type") <- "EMAX"
+  rlang::inform("Note updated EMAX model parameterization",
```
Comment (rich-payne) [inline R/model.R:215]: Can we update this message to something like "EMAX model parameterization has changed in dreamer version 4.   Please review the new model parameterization and update prior hyperparameters accordingly."

File: R/model.R [hunk 3]
```diff
@@ -245,11 +251,15 @@ model_exp <- function(
     sigma_b3 = sigma_b3,
     shape = shape,
     rate = rate,
+    bound = bound,
     w_prior = w_prior,
     longitudinal = longitudinal
   )
   class(mod) <- c("dreamer_exp", "dreamer_continuous")
   attr(mod, "type") <- "exponential"
+  rlang::inform("Note updated exponential model parameterization",
```
Comment (rich-payne) [inline R/model.R:260]: Update message similar to the EMAX one suggested.

File: R/model.R [hunk 4]
```diff
@@ -519,16 +528,21 @@ model_emax_binary <- function(
   )
   class(mod) <- c("dreamer_emax_binary", "dreamer_binary")
   attr(mod, "type") <- "EMAX"
+  rlang::inform("Note updated EMAX model parameterization",
```
Comment (rich-payne) [inline R/model.R:531]: Update message as suggested in the continuous EMAX model comment.

File: R/model.R [hunk 5]
```diff
@@ -519,16 +528,21 @@ model_emax_binary <- function(
...
+                .frequency_id = "3")
   return(mod)
 }
 
 #' @rdname model
+#' @param bound is the upper truncation bound of the exponential scale parameter \eqn{b_3}
 #' @section Exponential Binary:
-#'   \deqn{y \sim Binomial(n, f(d))}
-#'   \deqn{link(f(d)) = b_1 + b_2 * (exp(b_3 * d) - 1)}
+#'   \deqn{y \sim N(f(d), \sigma^2)}
```
Comment (rich-payne) [inline R/model.R:540]: This is incorrect -- we have a binary outcome so the likelihood is binomial.  Please revert.

File: R/model.R [hunk 6]
```diff
@@ -519,16 +528,21 @@ model_emax_binary <- function(
...
+#'   \deqn{f(d) = b_1 + b_2 * (exp(d / b_3) - 1)}
```
Comment (rich-payne) [inline R/model.R:541]: Please add link back in since this is a binary endpoint.

File: R/model.R [hunk 7]
```diff
@@ -548,12 +563,16 @@ model_exp_binary <- function(
...
     mu_b3 = mu_b3,
     sigma_b3 = sigma_b3,
+    bound = bound,
     link = link,
     w_prior = w_prior,
     longitudinal = longitudinal
   )
   class(mod) <- c("dreamer_exp_binary", "dreamer_binary")
   attr(mod, "type") <- "exponential"
+  rlang::inform("Note updated exponential model parameterization",
```
Comment (rich-payne) [inline R/model.R:573]: Please update message similar to that of the suggested EMAX model in other comment.

**Review by rich-payne — CHANGES_REQUESTED** [review]
Thank you for the updates!  Just a few more minor comments.

**Comment by mitchell-thomann** [comment]
Requested changes complete

**Review by rich-payne — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**